Mu2eG4/src/SConscript

```diff
@@ -18,10 +18,11 @@ helper=mu2e_helper(env)
 #
 
 g4inc    = os.environ['G4INCLUDE'] + '/Geant4'
+g4inc2   = os.environ['G4INCLUDE']
 g4libdir = os.environ['G4LIB']
 cryptoppinc = os.environ['CRYPTOPP_INC']
 cryptopplibdir = os.environ['CRYPTOPP_LIB']
-g4LibInc = [ '-L'+g4libdir, '-I'+g4inc, '-L'+cryptopplibdir, '-I'+cryptoppinc ]
+g4LibInc = [ '-L'+g4libdir, '-I'+g4inc, '-I'+g4inc2, '-L'+cryptopplibdir, '-I'+cryptoppinc ]
```

Comment:
Has G4 been modified so that we can modify all of our

#include "G4xxx.h"

to 

#include "Geant4/G4xxx.h"

If so, I propose that merge this PR and make that change in a separate PR - ack + perl should do it cleanly.